homeassistant/components/ps4/strings.json

```diff
@@ -19,21 +19,21 @@
         "data": {
```

Comment:
Please use the pin code reference in line 18.

Reply:
No, I actually meant to use it inside of the value e.g.
`"Enter your PlayStation 4 information. For '[%key:common::config_flow::data::pin%]', navigate to 'Settings' ...`

Reply:
Shouldn't this be information on docs side and not here?

Reply:
Well it refers to the docs as well for more information, so I would leave the information in this string.